Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -500,6 +506,52 @@ final class NoShippingPledgeViewController: UIViewController,
         self?.viewModel.inputs.scaFlowCompleted(with: status, error: error)
       }
   }
+
+  private func configurePledgeOverTimeUI(_ isPledgeOverTimeAllowed: Bool) {
+    guard isPledgeOverTimeAllowed else { return }
+
+    let collectionPlanStackView = UIStackView(frame: .zero)
+    collectionPlanStackView.axis = .vertical
+    collectionPlanStackView.spacing = Styles.grid(2)
+
+    let arrangedCollectionPlanSubviews = [
+      self.plansSectionLabel,
+      self.paymentPlansView
+    ]
+
+    arrangedCollectionPlanSubviews.forEach {
+      collectionPlanStackView.addArrangedSubview($0)
+    }
+
+    let collectionPlanSectionViews: [UIView] = [collectionPlanStackView]
+
+    let paymentStackView = UIStackView(frame: .zero)
+    paymentStackView.axis = .vertical
+    paymentStackView.spacing = Styles.grid(2)
+
+    let arrangedPaymentViewSubviews = [
+      self.paymentSectionLabel,
+      self.paymentMethodsViewController.view!
+    ]
+
+    arrangedPaymentViewSubviews.forEach {
+      paymentStackView.addArrangedSubview($0)
+    }
```

Comment:
Do we want the Payment section label to show regardless of whether PLOT is allowed? If we do, then we should probably move this logic into the existing `self.paymentMethodsSectionViews`  and also verify that it doesn't show when entering the "edit reward" flow when managing an existing backing.

Reply:
If this needs to be changed, more snapshots may also need to be updated. FYI

Reply:
As I know we want to show the sections labels just when PLOT is allowed, otherwise the current version of the checkout screen is shown.

I'm waiting for Alison's confirmation

Reply:
@scottkicks Margared answered this question and mentioned that the Payment label is not necessary to show when PLOT isn't allowed

[Figma thread](https://www.figma.com/design/vyTjzt9Tb67oHYUVbV2zKh?node-id=914-18814#1026951769)